child-logger.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+var defaultLevels = require('./default-levels.js');
+var makeLogMethod = require('./log-method');
+var errors = require('./errors');
+
+module.exports = ChildLogger;
+
+function ChildLogger(config) {
+    this.mainLogger = config.mainLogger;
+    this.path = config.path;
```

Comment:
Yep. I agree then we should throw at creation if we don't want double paths.

Do we want to consider the case that someone wants to use child logger with the same path more than once? That means they either need to manage the reference to the child logger themselves (could be acceptable) or the main logger can keep a reference so that it's accessible in the future. If the main logger keeps the reference it could cause a small memory leak.